tempest2-testing/src/main/kotlin/app/cash/tempest2/testing/TestDynamoDbServer.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.cash.tempest2.testing
+
+import com.google.common.util.concurrent.Service
+
+/**
+ * A DynamoDB test server running in-process or in a local Docker container.
+ */
+interface TestDynamoDbServer : Service {
```

Comment:
I’m wondering about the costs/benefits of sharing this between tempest 1 and 2. In theory the server set up doesn’t care which client API you’re using.

Reply:
I suspect it’s already too many modules, and what you have is nice in that you pick what stuff you’re already using and it just works.